src/Control/Effect/Pure.hs

```diff
@@ -38,6 +39,15 @@ instance Applicative PureC where
   (<*>) = coerce
   {-# INLINE (<*>) #-}
 
+  liftA2 = coerce
+  {-# INLINE liftA2 #-}
+
+  _ *> b = b
+  {-# INLINE (*>) #-}
+
+  a <* _ = a
+  {-# INLINE (<*) #-}
+
 instance Monad PureC where
   return = pure
```

Comment:
Should we add definitions of `<$` and `>>` as well?

Reply:
`<$` would probably get us some sort of benefit; I'll do that. `>>` is defined equal to `*>`, I believe.

Reply:
Will it inline through that definition?